server/fleet/datastore.go

```diff
@@ -1155,6 +1155,9 @@ type Datastore interface {
 	// serials.
 	UpdateDEPAssignProfileRetryPending(ctx context.Context, jobID uint, serials []string) error
 
+	// InsertMDMAppleDDMRequest inserts a DDM request.
+	InsertMDMAppleDDMRequest(ctx context.Context, hostUUID, message_type, json string) error
```

Comment:
Same nit for naming, just a heads-up that it would need to be updated here too.

Reply:
Done 👍